File: src/main/java/com/snackgame/server/applegame/controller/dto/CoordinateRequest.java
package com.snackgame.server.applegame.controller.dto;

import com.snackgame.server.applegame.domain.Coordinate;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;

@Getter
@NoArgsConstructor
@AllArgsConstructor
public class CoordinateRequest {

    private Integer y;
    private Integer x;

    public Coordinate toCoordinate() {
        return new Coordinate(y, x);
    }
}

File: src/main/java/com/snackgame/server/game/snackgame/core/service/dto/CoordinateRequest.kt
package com.snackgame.server.game.snackgame.core.service.dto

import com.snackgame.server.game.snackgame.core.domain.Coordinate

data class CoordinateRequest(
    var y: Int,
    var x: Int
){
    fun toCoordinate(): Coordinate =
        Coordinate(y, x)
}
